### DISCUSSION TITLE
Higress有交流群吗？

### DISCUSSION THREAD

**Discussion by zy0000001** [higress-group/higress#365] [Q&A] [answered]
把Higress交流群可以分享一下吗？官网的二维码已经失效了

**Comment by CH3CHO** [comment, marked as answer]
可以扫这个：

![image](https://github.com/alibaba/higress/assets/2909796/4ac8ae11-d1ff-4033-94ea-e431eadfcb28)

**Comment by zy0000001** [comment]
@CH3CHO 感谢

**Reply by CH3CHO** [reply]
官网二维码已更新
